README.md

```diff
@@ -143,3 +143,37 @@ search.addWidget(
 );
 ```
 
+### multipleChoiceList
+
+#### API
```

Comment:
I started documenting the option and thought "Well I am gonna invent my own jsdoc here :)"